src/help/guides/extension-development/function-definitions.md

```diff
@@ -82,19 +103,30 @@ async getChapters(mangaId: string): Promise<Chapter[]> {
     }
 
     // Return a list of all of the chapters discovered
-    return chapters
+    return chapters;
 }
 ```
 
 ## getChapterDetails
+
+### Method Signature
+
 `async getChapterDetails(mangaId: string, chapterId: string): Promise<ChapterDetails>`
 
-- A manga ID is defined by the source developer. However IDs are provided to Paperback in other functions, will be given to this function in the same format.
-- A ChapterID is defined by the [Get Chapters](#getChapters) method. However it is returned there, will be fed into this function.
+### Parameters
+
+| Parameter | Type | Description|
+|-----------|------|------------|
+|  `mangaId`  | String | The ID of a manga. The manga ID is provided from the other discovery functions, such as the home page and performed searches. The manga ID should be used to return information about the manga, such as the title and author. |
+| `chapterId` | String | The ID of a chapter. defined by the [Get Chapters](#getchapters) method. The chapter ID can only contain letters, numbers, dashes and underscores. |
 
-Given both a Manga and a Chapter ID, this function should populate a ChapterDetails object, which goes into further detail of what each chapter contains.
+### Returns
+
+Given both a Manga and a Chapter ID, this function should populate a [ChapterDetails](model-reference.md#ChapterDetails)
```

Comment:
As you said there was no difference, I would prefer:
```suggestion
Given both a Manga and a Chapter ID, this function should populate a [ChapterDetails](model-reference/#ChapterDetails)
```

Reply:
Actually, while I did say there was no difference, the latter breaks on my local testing system, so I actually think the first one is the way to go.

Reply:
I'm having better luck with what @FramboisePi has suggested. How does this break your local system?
I'm getting a strange navigation frame center with what's currently present here.

Reply:
For some reason, it's working now. I may have made a typo earlier causing it to not work, but regardless it seems to work now.